uint16 motor_id
int32 motor_pos